music/demos/visualizer.html

```diff
@@ -50,21 +51,87 @@ <h1>Visualizers</h1>
       <input id="tempoInput" value="100" type="range" min="20" max="240" value="120" step="1">
       <span id="tempoValue"></span>
     </span>
-  </section>
 
-  <h2>mm.PianoRollSVGVisualizer</h2>
-  <section>
+    <h3>mm.PianoRollSVGVisualizer</h3>
+    <div class="visualizer-container" id="container1">
+      <svg id="piano-roll-svg-viewer"></svg>
+    </div>
+
+    <h3>mm.StaffSVGVisualizer</h3>
     <div class="visualizer-container" id="container2">
-      <svg id="svg"></svg>
+      <div id="staff-svg-viewer"></div>
+    </div>
+
+    <h3>mm.PianoRollCanvasVisualizer</h3>
+    <div class="visualizer-container" id="container3">
+      <canvas id="piano-roll-canvas-viewer"></canvas>
     </div>
   </section>
 
-  <h2>mm.PianoRollCanvasVisualizer</h2>
   <section>
-    <div class="visualizer-container" id="container">
-      <canvas id="canvas"></canvas>
+    <h2>Advanced usage</h2>
```

Comment:
This entire paragraph feels like it belongs in the class docs for the SVGVisualizer, not the demo. For the demo I would maybe keep the multiple key signature example, since that seems different and maybe not straightforward?